src/main/io/serial.c

```diff
@@ -79,6 +79,7 @@ static uint8_t serialPortCount;
 const uint32_t baudRates[] = {0, 9600, 19200, 38400, 57600, 115200, 230400, 250000}; // see baudRate_e
 
 #define BAUD_RATE_COUNT (sizeof(baudRates) / sizeof(baudRates[0]))
+uint8_t numberOfBaudrateValues = BAUD_RATE_COUNT;
```

Comment:
Also slightly better way is to define function returning number of baudrates. With link time optimizations this function get's inlined, saving few bytes ...